joomla/joomla-cms
# Fixed small typos in months of Joomla core extension

I noticed that in the list of Extensions (**Extensions: Manage**) the "Multilingual Associations" has the wrong date "Januar 2017". 

Furthermore some "Joomla! Project" extensions have **abbreviated month names**, while the majority has full written month names. 

This small PR fixes that.

### Testing Instructions
The Extension: Manage shows the information from the #__extensions table. Upon installation the information from the XML is stored in the manifest_cache field of that #__extensions table.
To see the result of this patch, apply this patch before installing Joomla.
